WordPress/AbstractClassRestrictionsSniff.php

```diff
@@ -86,8 +86,8 @@ public function register() {
 	 *
 	 * @param int $stackPtr The position of the current token in the stack.
 	 *
-	 * @return int|void Integer stack pointer to skip forward or void to continue
-	 *                  normal file processing.
+	 * @return int Integer stack pointer to skip forward or void to continue
```

Comment:
My understanding is, that when a method could return a something or void, that `@return int|void`is allowed.

What's not allowed is the case of simply `@return void`.

Reply:
Maybe @DrewAPicture could weigh in on this.